### ISSUE TITLE
Creative Inventory ItemRenderer LocalCache out of bounds

### ISSUE THREAD

**Issue by dantaeusb** [mekanism/Mekanism#7744]
### Issue description

Quite sure it's some simple crash -- got it when tried to look for a torch in creative mode, see stack trace.

### Steps to reproduce

1. Play with Mekanism-1.19.2-10.3.8.477.jar and MekanismGenerators-1.19.2-10.3.8.477.jar
2. Go to creative inventory
3. Search for torch "tor"
4. Get hang on letter r and crash

### Minecraft version

1.19.2 (Latest)

### Forge version

43.2.0

### Mekanism version

10.3.8 (Latest)

### If a (crash)log is relevant for this issue, link it here: (It's almost always relevant)

https://gist.github.com/dantaeusb/3e7c3d80b734f77bb66e3e88329a7a30

**Comment by dantaeusb** [comment]
Interesting though - I am unable to reproduce that reliably. Changing texture packs was my assumption - but does not happen either. I'll continue trying to reproduce that, if I find the steps to reproduce reliably I will update the issue

**Comment by pupnewfster** [comment]
Can you reproduce this at all without optifine? Given yes I agree odds are optifine is doing something funky and breaking some rendering

**Comment by dantaeusb** [comment]
Yes, exploring the world of 1.16+ modding a bit more, I found things like Rubidium and Oculus which I tried to use instead of Optifine. Nothing like this happened without Optifine - I've tried to re-create a few worlds with reloading resource packs and shaders, and where the game with Optifine got weird, everything perfectly normal with Rubidium + Oculus. I would consider this an Optifine issue, then. Sorry for bothering you.

Thank you for your ongoing support on this mod!
